packages/vite-plugin-cloudflare/src/dev.ts

```diff
@@ -0,0 +1,22 @@
+import assert from 'node:assert';
+import { ROUTER_WORKER_NAME } from './constants';
+import type { ResolvedPluginConfig } from './plugin-config';
+import type { Miniflare } from 'miniflare';
+
+export function getDevEntryWorker(
+	resolvedPluginConfig: ResolvedPluginConfig,
+	miniflare: Miniflare,
+) {
+	const entryWorkerConfig =
+		resolvedPluginConfig.type === 'assets-only'
+			? resolvedPluginConfig.config
```

Comment:
nit: maybe we could add a comment mentioning why we pick the `resolvedPluginConfig.config` if the type is `assets-only`?